.github/workflows/build.yaml

```diff
@@ -374,3 +389,69 @@ jobs:
           asset_path: kube-linter-darwin.zip
           asset_name: kube-linter-darwin.zip
           asset_content_type: application/zip
+      - name: Upload sig Asset Linux
```

Comment:
It's probably good time to find beter action and clean it up or somehow generate it